app/configurations/realtimeUtils.js

```diff
@@ -73,6 +73,17 @@ export default {
 
     useFuzzyTripMatching: true, // DT-3473
   },
+  embark: {
+    mqttTopicResolver: walttiTopicResolver,
+
+    mqtt: 'wss://mqtt.waltti.fi/mqtt',
```

Comment:
FYI We will adapt this after merging.

Reply:
I don't know why I did it that way, but the URL is correct in the main config file.